### PR TITLE
Add workflows for Github

### PR DESCRIPTION
**Pull request by shubhagarwal1**
**Workflow to be implemeted**

- Auto Comment on Issue
- Auto Comment on PR
- Auto Comment on PR Merge
- Auto Label Issue
- Auto Label Issues
- Auto Label PRs
- Close Issues on PR Merge
- Close Old Issues
- Close Stale PRs
- Comment on Issue Close
- up for grabs

Hey @kom-senapati this could be buggy as each time one has to make the pull merge conflicts would increase 
Implement another workflow to update the contributors list in the README with every commit, encapsulated within detail-summary HTML tags.
i can think of another solution for this but please review and merge this

### CODE REVIEW

File: .github/workflows/auto_label_pr.yml
Comment (kom-senapati) [inline .github/workflows/auto_label_pr.yml:1]: This is duplicate of .github/workflows/auto_comment_pr_raise.yml. Remove this

File: .github/workflows/auto_lable_issue.yml
Comment (kom-senapati) [inline .github/workflows/auto_lable_issue.yml:1]: I don't want this workflow, instead edit issue templates for labels. Also no need for hacktoberfest and gssoc labels.

File: .github/workflows/close_issue_on_pr_merge.yml
Comment (kom-senapati) [inline .github/workflows/close_issue_on_pr_merge.yml:1]: Remove this workflow

File: .github/workflows/comment_iss-close.yml
Comment (kom-senapati) [inline .github/workflows/comment_iss-close.yml:1]: Remove this workflow

File: .github/workflows/auto_comment_pr_merge.yml
Comment (kom-senapati) [inline .github/workflows/auto_comment_pr_merge.yml:1]: Rename this to pr-workflow.yml

File: .github/workflows/auto_comment_pr_raise.yml
Comment (kom-senapati) [inline .github/workflows/auto_comment_pr_raise.yml:1]: Add this workflow part in .github/workflows/auto_comment_pr_merge.yml which will be renamed to pr-workflow.yml

File: .github/workflows/auto_comment_on_issue.yml
Comment (kom-senapati) [inline .github/workflows/auto_comment_on_issue.yml:1]: Rename this to issue-workflow.yml

**Review by kom-senapati — CHANGES_REQUESTED** [review]
Do these changes

**Comment by shubhagarwal1** [comment]
Hey @kom-senapati i have done the requested changes, please review the PR, also please assign the label =s.

**Review by kom-senapati — APPROVED** [review]

**Comment by kom-senapati** [comment]
LGTM

**Comment by kom-senapati** [comment]
Closes #185

**Pull request merged**